package.json

```diff
@@ -19,16 +19,16 @@
     "npm": ">=7.20.x <8.14"
   },
   "dependencies": {
-    "@bigcommerce/big-design": "^0.32.1",
+    "@bigcommerce/big-design": "^0.33.0",
     "@bigcommerce/big-design-theme": "^0.16.0",
-    "firebase": "^9.9.2",
+    "firebase": "^9.9.3",
```

Comment:
we might want to give this a slight bump as well - there's a `9.9.4` that's probably pretty safe

Reply:
I think with the carat `^` preceding the version, `npm i` automatically downloads the latest minor update — which in this case is `9.13.0` — and it's building fine for me locally so I think I could even bump it to that unless we want to be able to use `9.9.x`